src/core/lv2/Lv2Basics.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Lv2Basics.cpp - basic Lv2 functions
+ *
+ * Copyright (c) 2019-2019 Johannes Lorenz <j.git$$$lorenz-ho.me, $$$=@>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include "Lv2Basics.h"
+
+#ifdef LMMS_HAVE_LV2
+
+QString qStringFromPluginNode(const LilvPlugin* plug,
+	LilvNode* (*getFunc)(const LilvPlugin*))
+{
+	auto conv = lilv_node_as_string;
```

Comment:
I don't think it improves readability.

Reply:
I wanted to avoid breaking the next line, by making the function name shorter. But the other function `qStringFromPortName` doesn't do it that way, I'm changing it to your request and both functions now look similar.